src/modules/tas_studio/mod.rs

```diff
@@ -1870,6 +1887,13 @@ fn add_frame_bulk_hud_lines(text: &mut Vec<u8>, bulk: &FrameBulk) {
                 StrafeType::ConstYawspeed(yawspeed) => {
                     write!(text, "turn rate: {yawspeed:.0}").unwrap();
                 }
+                StrafeType::MaxAccelYawOffset(start, target, accel) => {
+                    // The values are so tiny that only this would make it sensible.
+                    let start = start * 100.;
+                    let target = target * 100.;
+                    let accel = accel * 100.;
+                    write!(text, "{start:.0} {target:.0} {accel:.2}").unwrap();
```

Comment:
```suggestion
                    write!(text, "from: {start:.0}, to: {target:.0}, accel: {accel:.2}").unwrap();
```

Reply:
I tried not having any text for this because it overflows the opaque background. I trust that anyone using this would understand what those numbers mean.

Reply:
Maybe write them on 3 lines? This will also allow not multiplying by 100. I'd rather this be understandable without looking at the code

Reply:
Hmm, I can leave it 1 line with 4 decimal numbers. It looks ok in worst case.

![image](https://github.com/user-attachments/assets/b1969d52-30cf-4749-9b66-bed49b6bab38)